Validate EDUST

edustdev opened the pull request:
# Validate EDUST

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/ExiledDustToken/status/1734131807191285877?t=2c8b3Sb_h7QQBNj2mbLi3A
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

Timoon21 commented on validated-tokens.csv:
```diff
@@ -1035,4 +1035,4 @@ Who,WHO,A8eYjpNQF6QYEE7DnM4F514MErFzV1pxVT69d7mdSsMk,9,https://bafybeiaahcxwjk6o
 Lamas Finance,LMF,LMFzmYL6y1FX8HsEmZ6yNKNzercBmtmpg2ZoLwuUboU,9,https://www.lamas.co/resource/lmf_token.png,true
 Rocky,ROCKY,4icEZCrEYNop2ZaMMCkRHaNzkt6xG9BpijMCQV7mpw6Z,9,https://i.ibb.co/X8r9531/rockylogo-transparents.png,true
-
+Exiled Dust Protocol,EDUST,21kbezaCxz9dG2NtXp7pW1hAmtKe6ykiJkeT1EiGXDyY,9,https://i.imgur.com/j4G1PZj.png,true
```
```suggestion
Exiled Dust Protocol,EDUST,21kbezaCxz9dG2NtXp7pW1hAmtKe6ykiJkeT1EiGXDyY,9,https://i.imgur.com/ExCfFFM.png,true
```

edustdev commented:
Just updated the url for the logo as I'd been advised that design wanted to update to the white background version officially to match token metadata onchain which I've also just updated.
